scrapers/VegasDreamworks.yml

```
@@ -0,0 +1,28 @@
+name: VegasDreamworks
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - toticos.com
+      - paradisegfs.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="post-title"]
+      Performers:
+        Name: //div[@class="single-model-head"]/a | //div[@class="update_bb_info"]/a
```

the parent is the same in both sites a simple
```suggestion
        Name: //div[@class="update_bb_info"]//a
```
should be ok